### DISCUSSION TITLE
Why does gatsby-focus-wrapper have tabindex = -1?

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [gatsbyjs/gatsby#36383] [Help] [answered]
## Question

- Why does gatsby-focus-wrapper have tabindex = -1?

![image](https://user-images.githubusercontent.com/27342882/184477177-dbf37ba3-14c1-4d8f-a628-60ad969ba44d.png)

## Description

Usually, if you click on a specific area and press the tab, you can access the tab to that area, but this is not the case with gatsby 

You may not understand what I'm saying, but I'll explain in detail.

### Example (Github)

![image](https://user-images.githubusercontent.com/27342882/184466432-0e245aa8-fc40-4327-882c-ad80f8f4362b.png)

Click on the area of ​​the photo on GitHub with the mouse and press the tab key to move the focus to the closest focusable area.

![image](https://user-images.githubusercontent.com/27342882/184466451-11c158ed-626b-4763-9696-040d291fc3a1.png)

### Gatsby

But for Gatsby, it always only goes to the top. Why is this value set?

**Comment by LekoArts** [comment, marked as answer]
It's to correctly announce when a user navigates to a new page: https://github.com/gatsbyjs/gatsby/issues/5581#issuecomment-509845632

Potential improvement in the future is that people could define the `<h1>` heading as announcement but for now that's how things are.
